int32 classID
string classname
int32[4] bbox